libtextsecure/account_manager.js

```diff
@@ -27,6 +27,8 @@
   function AccountManager(username, password) {
     // this.server = window.WebAPI.connect({ username, password });
     this.pending = Promise.resolve();
+    // set up mixpanel
+    window.mixpanel = new window.LokiMixpanelAPI();
```

Comment:
Might be better moving this up below `window.textsecure = window.textsecure || {};`.

`window.mixpanel = window.mixpanel || new window.LokiMixpanelAPI()`

thoughts?

Reply:
99% sure it's not created in account_manager.js at that point but can't hurt. Applied associated changes.